### PR TITLE
Add logging configuration example to the Configure inputs page

### PR DESCRIPTION
**Pull request by kilfoyle**
This adds a logging configuration example to the standalone agent [Configure inputs](https://www.elastic.co/guide/en/fleet/current/elastic-agent-input-configuration.html) page.

[DOCS PREVIEW](https://ingest-docs_bk_489.docs-preview.app.elstc.co/guide/en/fleet/master/elastic-agent-input-configuration.html)

Rel: #374
Closes: https://github.com/elastic/ingest-docs/issues/556

### CODE REVIEW

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (belimawr) [inline docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc:40]: This is not a generic type, this is the input type and needs to match one of the inputs supported by the Elastic-Agent. The supported input types are any Beat input or their aliases defined in the spec files, like [`filebeat.spec.yml`](https://github.com/elastic/elastic-agent/blob/698f7b99e5a26df9c2f311fec5ff792b4a11c4d6/specs/filebeat.spec.yml#L134-L137).

`data_stream.type` is a generic type describing the data that will be used to generate the index name.

The same applies to the other example down below.

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (belimawr) [inline docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc:23]: The flattened notation has not been released yet. We ended up reverting the Beats PR https://github.com/elastic/beats/pull/36610 due to a bug and now we're re-implementing it in the Elastic-Agent itself: https://github.com/elastic/elastic-agent/pull/3465.

We just need to make sure this version of the docs are released together with the feature.

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (belimawr) [inline docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc:70]: At the moment only the `filestream` input is supported in this simplified notation. In other words, it is only supported for log ingestion with the `filestream` input.

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (belimawr) [inline docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc:64]: I believe it would be interesting to also mention referring to the filestream documentation for other available options, like in the Metricbeat example.

**Review by belimawr — CHANGES_REQUESTED** [review]

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (kilfoyle) [reply docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc:64]: Good idea. Added via [a1fdb82](https://github.com/elastic/ingest-docs/pull/489/commits/a1fdb82dcd133f785eee59444350a206a72afdde)

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (kilfoyle) [reply docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc]: Good idea. Added via [a1fdb82](https://github.com/elastic/ingest-docs/pull/489/commits/a1fdb82dcd133f785eee59444350a206a72afdde)

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (kilfoyle) [reply docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc:23]: I added a note for myself in the description to wait until the Agent PR is merged.

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (kilfoyle) [reply docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc]: Nice! Thanks a lot for explaining this! To show what inputs are supported I made [these tables](https://ingest-docs_489.docs-preview.app.elstc.co/guide/en/fleet/master/elastic-agent-input-types.html). There were a few things I wasn't sure about though, in the third table (based on [filebeat.spec.yml](https://github.com/elastic/elastic-agent/blob/698f7b99e5a26df9c2f311fec5ff792b4a11c4d6/specs/filebeat.spec.yml)) and the fourth table (based on [heartbeat.spec.yml](https://github.com/elastic/elastic-agent/blob/698f7b99e5a26df9c2f311fec5ff792b4a11c4d6/specs/heartbeat.spec.yml)) so I left them blank with a `-`. If you have any suggestions for those that'd be great.

Comment (belimawr) [reply docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc]: Those tables look great @kilfoyle! I don't have much to add. I'm 95% sure the `docker` and `log/docker` are the `container` input.

I really like the idea of having an automated way to generate the docs and using the spec files looks great, they're a great source of truth for this.

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (belimawr) [inline docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc:40]: It might be just my developer's mind, but it feels a bit odd to me calling this "type of data to input". My understanding is that this is the input name, we use the keyword type but in the end of the day it's the name of the input.

Here it is just slightly different because we also support aliases, which makes thinking about "type of data" even more confusing for me.

Anyway, no need to change anything, I'm just commenting on how I perceive this, my mindset is likely very different than most of our users.

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (kilfoyle) [reply docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc]: Thanks! I've changed `docker` and `log/docker` to be described as aliases for `container`. I'm glad you like the automation idea!

File: docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc
Comment (kilfoyle) [reply docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc]: That makes sense to me! I've changed the description to `The name of the input.` And after I get the inputs list merged I will add a link to those so people know what values are available.

**Comment by kilfoyle** [comment]
I realized I should split this into separate PRs. That way, the [list of inputs PR](https://github.com/elastic/ingest-docs/pull/561) can be merged back into the current 8.10 docs (I've verified the list against the `8.10` and `main` versions of the Beats spec files).

This PR won't be backported since it describes the flattened `data_stream` notation being added via https://github.com/elastic/elastic-agent/pull/3465

**Review by karenzone — APPROVED** [review]
LGTM

**Comment by kilfoyle** [comment]
@belimawr I see that the flattened notation is merged now, so do you think it's okay to merge this docs change? Here's a [preview](https://ingest-docs_489.docs-preview.app.elstc.co/guide/en/fleet/master/elastic-agent-input-configuration.html), if don't mind taking a look.

**Comment by kilfoyle** [comment]
@elasticmachine run docs-build

**Review by belimawr — APPROVED** [review]

**Review by karenzone — APPROVED** [review]
LGTM

**Pull request merged**
